src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -148,6 +148,10 @@ export interface GetConceptsResponse {
   result: Concept[];
 }
 
+export interface GetConceptsHierarchyMapResponse {
```

Comment:
I would remove `Map` from this name as that's specific to the type being returned. In general we've been naming these objects like `ApiRequest` and `ApiResponse` where `Api` is the API name so this would be `GetConceptsHierarchyResponse`
```suggestion
export interface GetConceptsHierarchyResponse {
```